src/applications/vaos/utils/data.js

```diff
@@ -237,13 +243,18 @@ export function transformFormToCCRequest(state) {
 }
 
 export function transformFormToAppointment(state) {
+  const newAppointment = getNewAppointment(state);
   const data = getFormData(state);
   const clinic = getChosenClinicInfo(state);
-  const facility = getChosenFacilityInfo(state);
+  const parent = newAppointment.parentFacilities.find(
+    p => p.id === newAppointment.data.vaParent,
+  );
+  const siteId = getSiteIdFromOrganization(parent);
+  const { timezone = null } = siteId ? getTimezoneBySystemId(siteId) : {};
```

Comment:
When using the new endpoints, we no longer have the `institutionTimezone` available in legacyVAR.  We can derive this using the system id instead.